galaxy.yml

```diff
@@ -0,0 +1,16 @@
+namespace: community
+name: mysql
+version: 1.0.0
```

Comment:
```suggestion
version: 0.1.0
```
The first release should be 0.1.0. 1.0.0 should be done later when everything looks good.

Reply:
done 👍